Tests/SwiftParserTest/translated/TypealiasTests.swift

```diff
@@ -115,7 +118,10 @@ final class TypealiasTests: XCTestCase {
       diagnostics: [
         DiagnosticSpec(locationMarker: "1️⃣", message: "expected '=' in typealias declaration", fixIts: ["replace ':' with '='"]),
         DiagnosticSpec(locationMarker: "2️⃣", message: "expected type in typealias declaration", fixIts: ["insert type"]),
-      ]
+      ],
+      fixedSource: """
+        typealias Recovery2 =<#type#>
```

Comment:
`=` should have balanced whitespace.

Reply:
https://github.com/apple/swift-syntax/issues/1609